doc/release/RELEASE-NOTES.md

```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1105](https://opensupplyhub.atlassian.net/browse/OSDEV-1105) - Contribution. Allow commas in list name and update error message.
 * [OSDEV-272](https://opensupplyhub.atlassian.net/browse/OSDEV-272) - Facility Claims Page. Implement ascending/descending and alphabetic sort on FE. Applied proper sorting for lower case/upper case/accented strings.
 * [OSDEV-1036](https://opensupplyhub.atlassian.net/browse/OSDEV-1036) - Claims. Add a sortable "claim decision" column to claims admin page.
+* [OSDEV-1083](https://opensupplyhub.atlassian.net/browse/OSDEV-1083) - Enable Password Visibility on Chrome Browser.
```

Comment:
It looks like you just copied the ticket name. Please provide a more descriptive explanation of what has been done.

Your changes will be in release 1.17. So, please add a new section, "Release 1.17.0," to the release notes from `RELEASE-NOTES-TEMPLATE.md `and move the description for OSDEV-1083 there.

Don't forget to add the correct `Fix version` to the ticket.

Reply:
Please add the correct `Fix version` to the ticket that I mentioned above.